README.md

```diff
@@ -25,20 +25,35 @@ pip install synapse-token-authenticator
 ## Configuration
 Here are the available configuration options:
 ```yaml
-# provide only one of secret, keyfile
-secret: symetrical secret
-keyfile: path to asymetrical keyfile
-
-# Algorithm of the tokens, defaults to HS512
-#algorithm: HS512
-# Allow registration of new users using these tokens, defaults to false
-#allow_registration: false
-# Require tokens to have an expiracy set, defaults to true
-#require_expiracy: true
+jwt:
+  # provide only one of secret, keyfile
+  secret: symetrical secret
+  keyfile: path to asymetrical keyfile
+
+  # Algorithm of the tokens, defaults to HS512 (optional)
+  algorithm: HS512
+  # Allow registration of new users, defaults to false (optional)
+  allow_registration: false
+  # Require tokens to have an expiry set, defaults to true (optional)
+  require_expiry: true
+oidc:
+  issuer: "https://idp.example.com"
+  client_id: "<IDP client id>"
+  client_secret: "<IDP client secret>"
+  # Zitadel Organization ID, used for masking. (Optional)
+  organization_id: 1234
+  # Zitadel Project ID, used for validating the audience of the returned token.
+  project_id: 5678
+  # Limits access to specified clients. Allows any client if not set (optional)
+  allowed_client_ids: ['2897827328738@project_name']
+  # Allow registration of new users, defaults to false (optional)
+  allow_registration: false
 ```
-It is recommended to have `require_expiracy` set to `true` (default). As for `allow_registration`, it depends on usecase: If you only want to be able to log in *existing* users, leave it at `false` (default). If nonexistant users should be simply registered upon hitting the login endpoint, set it to `true`.
+It is recommended to have `require_expiry` set to `true` (default). As for `allow_registration`, it depends on usecase: If you only want to be able to log in *existing* users, leave it at `false` (default). If nonexistant users should be simply registered upon hitting the login endpoint, set it to `true`.
```

Comment:
When we talked about fixing this typo, we were under the impression that the JWT authenticator would remain in use. This is no longer the case, really, this flow is planned to replace any existing JWT auth. With this in mind, I think we should keep the misspelled name, as it won't remain in use for long anyway and the effort of adjusting configurations seems unnecessary.

Reply:
Didn't we agree that since the config would need to be adjusted to add new keys anyway, it would be fine to also change another key? It can easily be reverted if necessary, I'm just not sure if it's a good idea.